tools/etl/tg-jdbc-driver/tg-jdbc-examples/src/main/java/com/tigergraph/jdbc/examples/Builtins.java

```diff
@@ -27,21 +27,21 @@ public static void main( String[] args ) throws SQLException {
     properties.put("password", "tigergraph");
 
     /**
-     * Specify the graph name, especially when multi-graph is enabled.
-     */
-    properties.put("graph", "gsql_demo");
-
-    /**
-     * Only accept 3 parameters: IP address, port and debug.
+     * Only accept 4 parameters: IP address, port, debug and graph name.
      * Enable debug when the third parameter's value is larger than 0.
      */
-    if (args.length == 3) {
+    if (args.length == 4) {
       ipAddr = args[0];
       port = Integer.valueOf(args[1]);
+
       properties.put("debug", args[2]);
+
+      // Specify the graph name, especially when multi-graph is enabled.
+      properties.put("graph", args[3]);
     }
 
     /**
+     * Running on TigerGraph V3.4
```

Comment:
Please remove "Running on TigerGraph V3.4", it's supposed to be compatible with all versions.